kvbc/src/v4blockchain/detail/blockchain.cpp

```diff
@@ -24,6 +24,9 @@ Blockchain::Blockchain(const std::shared_ptr<concord::storage::rocksdb::NativeCl
   if (native_client->createColumnFamilyIfNotExisting(v4blockchain::detail::BLOCKS_CF)) {
     LOG_INFO(V4_BLOCK_LOG, "Created [" << v4blockchain::detail::BLOCKS_CF << "] column family for the main blockchain");
   }
+  if (native_client_->createColumnFamilyIfNotExisting(v4blockchain::detail::MISC_CF)) {
```

Comment:
2 things:
1) nit: consider  to have a more meaningful name? I  can't  really understand much from this name unless I go and search what is stored under it.
e.g METADATA_CF or ADDITIONAL_INFO_CF

2) Why do you LOG_PRINT outside the function? Why not print inside createColumnFamilyIfNotExisting if created? Do you want to avoid to many prints? for that toy can also pass a flag e.g print_on_creation with default set to false.

Reply:
```createColumnFamilyIfNotExisting``` is an API call; I prefer leaving it as clean as possible.